spec/features/practice_spec.rb

```diff
@@ -184,13 +184,6 @@
 
   describe 'Next Steps' do
```

Comment:
I think we can remove this entire Next Steps test section

Reply:
done!